kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/StampViewModel2.java

```diff
@@ -222,6 +222,7 @@ public StampViewModel2 save() {
         State status = getValue(STATUS);
         EntityFacade module = getValue(MODULE);
         EntityFacade path = getValue(PATH);
+        EntityFacade author = getValue(AUTHOR);
```

Comment:
Instead of getting the author from the view model, couldn't you just get it from the `ViewProperties`.
Transactions would always write using the logged in user.

Reply:
```java
ConceptFacade loginUser = viewProperties.nodeView().editCoordinate().getAuthorForChanges();
```